src/System.Drawing.Primitives/src/System/Drawing/Rectangle.cs

```diff
@@ -262,6 +254,8 @@ public override bool Equals(object obj)
             (X <= rect.X) && (rect.X + rect.Width <= X + Width) &&
             (Y <= rect.Y) && (rect.Y + rect.Height <= Y + Height);
 
+        public bool Equals(Rectangle other) => this == other;
```

Comment:
Nit: any reason this is down here rather than up with the other Equals as it is on other types?